test/integration/trafficsplit/trafficsplit_test.go

```diff
@@ -69,11 +69,11 @@ func parseStatTsRow(out string, expectedRowCount, expectedColumnCount int) (map[
 
 func statTrafficSplit(from string, ns string) (map[string]*statTsRow, error) {
 	cmd := []string{"stat", "ts", "--from", from, "--namespace", ns, "-t", "30s", "--unmeshed"}
-	stdOut, _, err := TestHelper.LinkerdRun(cmd...)
```

Comment:
Note that this was the case that triggered this PR -- we dropped the stderr silently. But now it's impossible to do so ;)